task/04-arrays-tasks.js

```diff
@@ -304,9 +316,12 @@ function get3TopItems(arr) {
  *   [ 1, '2' ] => 1
  */
 function getPositivesCount(arr) {
-   throw new Error('Not implemented');
+   //return arr.filter(element => element > 0).length;
+   let arr_rez = [];
+   arr_rez = arr.filter(function (v) { return v > 0 & typeof v !== 'string'; });
```

Comment:
Please, use 'logical and'  `&&`

https://stackoverflow.com/questions/7310109/whats-the-difference-between-and-in-javascript#:~:text=To%20determine%20whether%20two%20boolean,to%20check%20the%20second%20value.

try in chrome console 
`console.log(true && 5)`
`console.log(true & 5)`